docs/developers/ROADMAP_0_3.md

```diff
@@ -0,0 +1,76 @@
+# Roadmap for 0.3 series of releases - April 2020.
+
+The napari roadmap captures current development priorities within the project and should serve as a guide for napari core developers, to encourage and inspire contributors, and to provide insights to external developers who are interested in building for the napari ecosystem. For more details on what this document is and is not, see the [about this document section](#about-this-document).
+
+The [mission](MISSION_AND_VALUES.md#our-mission) of napari is to be a foundational multi-dimensional image viewer for Python and provide graphical user interface (GUI) access to image analysis tools from the Scientific Python ecosystem for scientists across domains and levels of coding experience. To work towards this mission, we have set the following three high-level priorities to guide us over the upcoming months:
+
+- Make the **data viewing and annotation** capabilities **bug-free and fast**.
+
+- Make a **downloadable application** with reader / writer plugin management.
+
+- Make accessible documentation, tutorials, and demos.
+
+Once the above goals are met, we will develop napari's capabilities for image processing and analysis. We are prioritizing the robustness and polish of the core viewer before adding advanced features or support for functional or interactive plugins to ensure that plugin development will happen against a solid foundation, and because we have repeatedly encountered biologists and other scientists who have trouble even looking at their data, and annotating it. We are prioritizing the downloadable application with reader / writer plugin management so that we can start getting feedback from non-coding users to better understand their needs.
+
+### Bug-free and fast data viewing and annotation
```

Comment:
heading levels should preferably only increment by one level at a time (this is sometimes problematic in sphinx)

Reply:
done